backend/apps/users/models.py

```
@@ -0,0 +1,117 @@
+    address = models.TextField()
+    country = models.CharField(max_length=255, default='Bangladesh')
+    dob = models.DateField(default='2000-01-01')
+    image = CloudinaryField('image', null=True, blank=True)
+    gender = models.CharField(max_length=10, choices=gender)
+    blood_group = models.CharField(max_length=5, choices=Blood_Group)
+    blood_pressure = models.CharField(max_length=10)
+    height = models.DecimalField(max_digits=5, decimal_places=2, null=True, blank=True)
+    weight = models.DecimalField(max_digits=5, decimal_places=2, null=True, blank=True)
+    bmi = models.CharField(max_length=10, blank=True, default="")
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)